index.html

```diff
@@ -550,6 +550,138 @@
 template: inverse
 
 ## Serwer danych **Django**
+---
```

Comment:
you did't write anythink about the model (where could be some simple image with the classes relations etc. to show the base concept of data stored on server)